chirpr.py

```diff
@@ -119,9 +128,37 @@ def chirps():
     return render_template('admin/chirps.html', chirps=chirp_list)
     
     
+@app.route('/chirps/<handle>')
+def chirps_hn(handle):
+    uid = db_access.get_id(handle)
+    return redirect('%s?filter=%s'%(url_for('chirp'), handle))
```

Comment:
OK, I don't like this redirect. Rather than building a string, why not pass the `handle` into `url_for`? I think that can work. Also, I'm not sure what `chirps_hn` is supposed to be. Names matter